cmd/main.go

```diff
@@ -165,19 +165,30 @@ func main() {
 	}
 }
 
+func getSiteConfigNamespace(log logr.Logger) string {
+	namespace := os.Getenv("POD_NAMESPACE")
+	if namespace == "" {
+		log.Info("Failed to retrieve SiteConfig namespace, using default", "default", defaultSiteConfigNamespace)
+		return defaultSiteConfigNamespace
```

Comment:
I think this should be fatal. Do you think there's any reason to maintain a default?

Reply:
tbh - probably no significant reason to maintain a default :D

Reply:
Should we just `panic()`?

Reply:
Changed to `os.Exit(1)`